data/json/monstergroups/wilderness.json

```diff
@@ -223,7 +222,8 @@
         "weight": 1,
         "cost_multiplier": 15,
         "pack_size": [ 2, 5 ],
-        "conditions": [ "AUTUMN", "WINTER" ]
+        "conditions": [ "AUTUMN", "WINTER" ],
```

Comment:
```suggestion
```
Please remove this line, it's duplicative in intent with what you're doing and what you're doing is better